### ISSUE TITLE
Moquette-parent v0.14 is not in jcenter

### ISSUE THREAD

**Issue by hylkevds** [moquette-io/moquette#565]
The pom.xml of Moquette-broker specifies that it has a parent, moquette-parent with version number 0.14
https://jcenter.bintray.com/io/moquette/moquette-broker/0.14/:moquette-broker-0.14.pom
```
    <parent>
        <relativePath>../</relativePath>
        <artifactId>moquette-parent</artifactId>
        <groupId>io.moquette</groupId>
        <version>0.14</version>
    </parent>
```
However, moquette-parent version 0.14 is not available in jcenter:
https://jcenter.bintray.com/io/moquette/moquette-parent/

This makes any build fail, that depends on 0.14

**Comment by hylkevds** [comment]
To test yourself, just delete moquette from your maven cache, and try to build anything that depends on 0.14
:)

**Comment by hylkevds** [comment]
Now that [JCenter is being shut down by JFrog](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/), moquette will also have to find a new home.
I just did a quick check, and moquette does not depend on anything that is not in Maven Central, so it would make sense to move moquette there. One caveat would be that [Maven Central requires that you "control" the domain in your group id](https://central.sonatype.org/pages/producers.html). Though there are plenty of io.xxx entries that point to github/xxx-io
The rest of the setup is not that hard, I've just converted most of our projects to Maven Central.

Or do you have a different repository in mind?

**Comment by hylkevds** [comment]
This was also fixed with commit [3f54b59](https://github.com/moquette-io/moquette/commit/3f54b59688d237896145ddfd00b1aaf3afbf0489)
